_drafts/2017-08-23-The-value-of-a-digital-workplace.md

```diff
@@ -0,0 +1,28 @@
+ 
+---
+
+Here are some of the hidden benefits.
+
+**Streamlining**. If you want to use your digital workplace to its best advantage, you need to reconsider and optimize your processes. This means redesigning workflows, identifying and removing unnecessary steps or entire processes, resolving issues that cause productivity bottlenecks – and automating all that is possible. At the end of this process it might turn out that a considerable amount of valuable human work time is freed up to do more of the creative object.
+
+**User experience**. A good digital workplace is tailored to the needs of its users, and not the users are required to bend around the (in)capabilities of just another new environment. If user experience is a top priority right from the beginning of the design phase, it will naturally result in happy users. And a happy employee is a more productive employee. To enhance user experience you need simple applications with intuitive interfaces that provide exactly the data necessary for the completion of the task the employee is required to complete.
+
+![value_of_dw_userexp](/img/posts/value_of_dw_userexp.jpg)
+
+**Cross-platform**. A well-designed digital workplace enables user access from various devices, such as PCs, notebooks, tablets and mobiles. Can you imagine a situation when an employee has an urgent task to finish, her workday is already over and at the same time, has family or other commitments as well? A lifelike situation, isn’t it? Instead of being bound to the device located in the office, with a digital workplace solution this employee can finish the task on-the-go from a mobile device, or later from home. This means less stress and less stress means more relaxed and thus more productive employees. But this is not the only advantage of the cross-platform design. Lengthy travel times of business trips can be turned into productive time and field results can be entered into the system and made available to everyone right away.
+
+**Data access**. If all data are stored digitally, they are easily searchable, shareable and quickly available anywhere, anytime, to anyone (depending on access privileges, of course). In a digital workplace data is created digitally – meaning that you always have access to the most recent data right away. This has clear advantages in decision making, when using outdated information might lead to dangerous, even business-critical results. But the advantages don’t stop here. If you have digital data, it means you have a digital archive as well. This eliminates the need to store paper copies, which make your business considerably cheaper, leaner and swifter, not to mention more sustainable and eco-friendly.
```

Comment:
This eliminates the need to store paper copies, **that makes** your business considerably cheaper, leaner and swifter, not to mention more sustainable and eco-friendly.

Reply:
ok